spec/lib/ldap_authenticatable_tiny_spec.rb

```diff
@@ -107,42 +107,58 @@ def test_options(login, password)
 
   describe '#is_authorized_by_ldap? calls Net::LDAP correctly' do
     before { set_env }
+    let!(:pals) { TestPals.create! }
 
     it 'returns false and locks for invalid credentials' do
-      mock_ldap = instance_double(Net::LDAP)
-      expect(mock_ldap).to receive(:bind) { false }
-      expect(mock_ldap).to receive(:get_operation_result) { 'result-error-message' }
-      expect(Net::LDAP).to receive(:new).and_return(mock_ldap)
-
       strategy = test_strategy
-      expect(strategy.send(:is_authorized_by_ldap?, 'foo', 'bar')).to eq false
+
+      args = [
+        :is_authorized_by_ldap?,
+        TestPalsMockLDAP,
+        'uri@demo.studentinsights.org',
+        'bar'
+      ]
+
+      expect(strategy.send(*args)).to eq false
```

Comment:
This seems tricksy to me versus saying it plain, is this to avoid long line length?

Reply:
Line length but also making it easy for the reviewer to see which arguments are changing for each test case.

Reply:
I'd vote for finding another way to do this that doesn't involve `send(*args)` where the code hides what method is being called.  Another way to make it easier to see the arguments on each line might be to use the splat inline:

```
expect(strategy.is_authorized_by_ldap?(*[
  MockLDAP,
  'uri@demo.studentinsights.org',
  'bar'
])).to eq false

Reply:
Got it!  Yeah `send` let's you call private methods in test.  The thing I'm trying to comment on is that using send with a variable together like `send(*args)` means you can't tell reading that code what method is being called.  I agree you should 🚢 what you think makes sense 👍